packages/lib-user/src/components/GroupStats/helpers/getHeaderItems.js

```diff
@@ -46,14 +46,14 @@ function getHeaderItems({
         label='Share Group'
         message='Group Link Copied!'
         primaryItem={true}
-        textToCopy={`https://www.zooniverse.org/groups/${group.id}`}
+        textToCopy={`https://fe-root.preview.zooniverse.org/groups/${group.id}`}
```

Comment:
I personally like seeing base url's as `.env` variable that get interpolated, but in this case it probably makes more sense to create const variable at the top so that it's not hardcoded in multiple locations. Something like:

`const BASE_URL = 'https://fe-root.preview.zooniverse.org';`

... then the code becomes ``textToCopy={`${BASE_URL}/groups/${group.id}\`}``

Reply:
Excellent suggestion, refactored accordingly in 26bd531